prow/config/config.go

```diff
@@ -262,18 +262,19 @@ type Branding struct {
 
 // Load loads and parses the config at path.
 func Load(prowConfig, jobConfig string) (*Config, error) {
-	nc, err := loadHelper(prowConfig, jobConfig)
+	nc, err := loadConfig(prowConfig, jobConfig)
 	if err != nil {
 		return nil, err
 	}
 
-	if err := nc.finalizeDecoration(); err != nil {
+	if err := nc.validateJobConfig(); err != nil {
```

Comment:
hmmm is validate really the best name for something that mutates the config? :confused:

Reply:
Yeah I remember that but, let's stop doing that.
If I didn't already know about it here I'd find a `validate` call mutating anything pretty surprising...